django_notify/urls.py

```diff
@@ -13,8 +13,8 @@
     url('^goto/$', views.goto, name='goto_base', kwargs={}),
 ]
 
-def get_pattern(app_name="notify", namespace="notify"):
+def get_pattern(app_name="notify"):
```

Comment:
The changes here and in `wiki/urls.py` will require some minor changes in edx-platform's `lms/urls.py`, so keep that in mind when updating edx-platform to use this code.

Reply:
@jmbowman I made a testing platform [PR](https://github.com/edx/edx-platform/pull/22742) just to check the failures.